src/proof/resolution_bitvector_proof.h

```diff
@@ -0,0 +1,102 @@
+/*********************                                                        */
+/*! \file resolution_bitvector_proof.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Liana Hadarean, Mathias Preiner, Guy Katz
+ ** This file is part of the CVC4 project.
+ ** Copyright (c) 2009-2018 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved.  See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** \brief Bitvector proof
+ **
+ ** Bitvector proof
+ **/
+
+#include "cvc4_private.h"
+
+#ifndef __CVC4__RESOLUTION__BITVECTOR__PROOF_H
+#define __CVC4__RESOLUTION__BITVECTOR__PROOF_H
+
+#include <iostream>
+#include <sstream>
+
+#include "expr/expr.h"
+#include "proof/bitvector_proof.h"
+#include "proof/theory_proof.h"
+#include "prop/bvminisat/core/Solver.h"
+
+
+namespace CVC4 {
+
+namespace prop {
+class CnfStream;
+} /* namespace CVC4::prop */
+
+namespace theory {
+namespace bv {
+class TheoryBV;
+template <class T> class TBitblaster;
+} /* namespace CVC4::theory::bv */
+} /* namespace CVC4::theory */
+
+class CnfProof;
+} /* namespace CVC4 */
+
+namespace CVC4 {
+
+template <class Solver> class TSatProof;
+typedef TSatProof< CVC4::BVMinisat::Solver> BVSatProof;
+
+class ResolutionBitVectorProof : public BitVectorProof {
```

Comment:
Please add a comment for every class, method, and member variable in the header file.

Reply:
It's easy for me to write comments for members that I've come to understand in the course of implementing DRAT support (most of these are in `BitVectorProof`), and I'll go through and do so.

However,  most of the members in `ResolutionBitVectorProof` are not connected to DRAT, and I haven't looked into what they do. I wouldn't be able to write good comments for them right now.

If needed, I'm happy to look into this, but it would be a new task.

Reply:
Yeah, I think that it is totally fine to just add comments to the ones you know and address the other ones later.